packages/api/src/command/medical/patient/create-medical-record.ts

```diff
@@ -74,10 +85,40 @@ export async function getMedicalRecordSummary({
   const pdfIsValid = conversionType === "pdf" && pdfCreatedAt;
   const htmlIsValid = conversionType === "html" && htmlCreatedAt;
 
+  let s3BucketName = bucketName;
+  let s3FileKey;
+
+  if (Config.isSandbox()) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    s3BucketName = Config.getSandboxSeedBucketName()!;
+    if (pdfIsValid || htmlIsValid) {
+      const patientName = await getSandboxPatientName(patientId, cxId);
+      s3FileKey = createSandboxMRSummaryFileName(patientName, conversionType);
+      const url = await getSignedURL({ bucketName: s3BucketName, fileName: s3FileKey });
+      return url;
+    }
+  }
+
   if (pdfIsValid || htmlIsValid) {
-    const s3FileKey = createMRSummaryFileName(cxId, patientId, conversionType);
-    const url = await s3Utils.getSignedUrl({ bucketName, fileName: s3FileKey });
+    s3FileKey = createMRSummaryFileName(cxId, patientId, conversionType);
+    const url = await s3Utils.getSignedUrl({ bucketName: s3BucketName, fileName: s3FileKey });
     return url;
   }
   return;
 }
+
+async function getSandboxFileNames(patientId: string, cxId: string): Promise<[string, string]> {
+  const firstName = await getSandboxPatientName(patientId, cxId);
+  const s3HtmlSandboxKey = createSandboxMRSummaryFileName(firstName, "html");
+  const s3PdfSandboxKey = createSandboxMRSummaryFileName(firstName, "pdf");
+  return [s3HtmlSandboxKey, s3PdfSandboxKey];
+}
+
+async function getSandboxPatientName(patientId: string, cxId: string) {
+  const patient = await getPatient({ id: patientId, cxId });
+  return patient ? patient.data.firstName.toLowerCase() : "jane";
+}
+
+function createSandboxMRSummaryFileName(firstName: string, extension: "pdf" | "html"): string {
+  return extension === "pdf" ? `${firstName}_MR.html.pdf` : `${firstName}_MR.html`;
+}
```

Comment:
These show up twice in this PR on diff places. Can we make those a shared function? Especially the default patient name for sandbox, since the other one likely can be replaced by calling `createMRSummaryFileName`.